components/bl-stepper-component/src/index.js

```diff
@@ -42,13 +42,15 @@ export default function Stepper({ component, eventHandlers }) {
     setSteps(steps.map(step => ({ ...step, completed: false })));
   };
 
-  component.setStep = stepNumber => {
-    setCurrentStep(stepNumber - 1);
-    setSteps(steps.map((step, index) => {
-      const completed = index < stepNumber - 1;
-
-      return { ...step, completed };
-    }));
+  component.setStep = nextStep => {
+    if (inRange(nextStep, steps.length + 1)) {
```

Comment:
```suggestion
    if (inRange(nextStep, steps)) {
```

Move works with length into function

Reply:
Or, what if we rewrite it without function like

```
if( steps.length + 1 > nextStep > 0) {...}
```